build.gradle.kts

```diff
@@ -81,14 +81,13 @@ repositories {
 // The provided example uses Apache Commons Math. Replace or add as needed.
 dependencies {
     // resolve Processing core
-    compileOnly("com.github.micycle1:processing-core-4:4.3.1")
+    compileOnly(group = "com.github.micycle1", name = "processing-core-4", version = "4.3.1")
```

Comment:
Even though it works and I like it, it is very non-standard to do it this way

Reply:
Gotcha. I changed it to this because while I was developing the example library, I had to debug wrong punctuation in this declaration, and this way avoids that error pathway. I think people who know otherwise then can change as they wish.